src/main/resources/templates/rocket.ftl

```diff
@@ -0,0 +1,12 @@
+<#compress>
+    **${results}:**
+    **-${environment}:** ${env}
+    **-${comment}:** ${comm}
+    **-${totalScenarios}:** ${total}
+    <#if passed != 0 > **-${totalPassed}:** ${passed} </#if>
+    <#if failed != 0 > **-${totalFailed}:** ${failed} </#if>
+    <#if broken != 0 > **-${totalBroken}:** ${broken} </#if>
+    <#if unknown != 0 > **-${totalUnknown}:** ${unknown} </#if>
+    <#if skipped != 0 > **-${totalSkipped}:** ${skipped} </#if>
+    <#if reportLink??>**${reportAvailableAtLink}:** ${reportLink}</#if>
+</#compress>
```

Comment:
is there any reason to have additional template?

Reply:
with default template have some issues 
<img width="421" alt="image" src="https://github.com/qa-guru/allure-notifications/assets/36734883/ff8ef01e-3a69-40c9-845e-41ff6948c5b8">